aws-sqs.yml

```diff
@@ -34,6 +34,14 @@ provision:
       type: string
       details: AWS secret key
       default: ${config("aws.secret_access_key")}
+    - field_name: dlq_arn
+      type: string
+      details: ARN of the Dead Letter Queue. If provided, configures redrive_policy for the queue.
+      default: ""
+    - field_name: max_receive_count
```

Comment:
We did at one point talk about prefixing DLQ field names so this could be `dlq_max_receive_count`.

Reply:
I'm going to change it now to establish the style for the next changes. Thank you @blgm